src/server/visual-diff-plugin.js

```diff
@@ -12,6 +20,39 @@ async function checkFileExists(fileName) {
 	}
 }
 
+async function clearDir(updateGoldens, path) {
+	if (updateGoldens) {
+		await rm(path, { force: true, recursive: true });
+	} else {
+		await rm(join(path, PATHS.FAIL), { force: true, recursive: true });
+		await rm(join(path, PATHS.PASS), { force: true, recursive: true });
+		await rm(join(path, 'report.html'), { force: true });
```

Comment:
Can these be done concurrently or does it error out?
```suggestion
		await Promise.all([
			rm(join(path, PATHS.FAIL), { force: true, recursive: true }),
			rm(join(path, PATHS.PASS), { force: true, recursive: true }),
			rm(join(path, 'report.html'), { force: true })
		]);
```

Reply:
Seems to work!